SU-SWS/stanford_components
# Earth 247 degree transition

# READY FOR REVIEW

# Summary
- Refactors responsive grid on Section Expandable Header and adds transition for the Expandable Cards.

# Needed By (Date)
- Next Sprint (7/21)

# Steps to Test

1. View Section Expandable Header organism here: /patterns/section_expandable_banner
2. Click the [+] buttons on the expandable cards
3. You should see a fadeIn animation

# Affected versions or modules
- Does this PR impact any particular module, version, or pull request? - No

# Associated Issues and/or People
- JIRA ticket EARTH-247
- Other PRs: [Matson PR #61](https://github.com/stanford-earth/matson/pull/61)
